azure-service-bus-management/1.0/modules/ROOT/pages/azure-service-bus-management-connector-examples.adoc

```diff
@@ -0,0 +1,246 @@
+= Azure Service Bus Management 1.0 Examples - Mule 4
+
+The examples in this section show how to get information about Azure Service Bus queues and how to create an Azure Service Bus topic.
+These examples use variables for some field values. You can either:
+
+* Replace the variables with their values in the code
+* Provide the values for each variable in a properties file and then refer to that file from the connector configuration
++
+For more information about using a properties file, see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders]
+
+== Get Queue Information Example
```

Comment:
In Kim's edit, she had me put Example first in the header title

Reply:
I don't agree but I'll change it.